2025-03-09 20:57:28,051 INFO: D√©marrage de l'application [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:71]
2025-03-09 20:58:45,398 INFO: D√©marrage de l'application [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:71]
2025-03-09 21:01:18,228 INFO: D√©marrage de l'application [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:71]
2025-03-09 21:01:51,293 INFO: Connexion r√©ussie pour l'utilisateur: nadim.benkrouidem@gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:115]
2025-03-09 21:02:04,742 INFO: D√©but de l'upload [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:188]
2025-03-09 21:02:04,746 INFO: Upload vers UserImg/nadim.benkrouidem_at_gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:206]
2025-03-09 21:02:05,152 INFO: Analyse de l'image avec l'IA [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:222]
2025-03-09 21:02:05,152 ERROR: Erreur lors de l'analyse de l'image: 'NoneType' object has no attribute 'predict' [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:243]
2025-03-09 21:02:06,988 INFO: D√©but de l'upload [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:188]
2025-03-09 21:02:06,990 INFO: Upload vers UserImg/nadim.benkrouidem_at_gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:206]
2025-03-09 21:02:07,360 INFO: Analyse de l'image avec l'IA [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:222]
2025-03-09 21:02:07,361 ERROR: Erreur lors de l'analyse de l'image: 'NoneType' object has no attribute 'predict' [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:243]
2025-03-09 21:02:08,715 INFO: D√©but de l'upload [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:188]
2025-03-09 21:02:08,718 INFO: Upload vers UserImg/nadim.benkrouidem_at_gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:206]
2025-03-09 21:02:08,910 INFO: Analyse de l'image avec l'IA [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:222]
2025-03-09 21:02:08,911 ERROR: Erreur lors de l'analyse de l'image: 'NoneType' object has no attribute 'predict' [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:243]
2025-03-09 21:06:56,044 INFO: D√©marrage de l'application [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:71]
2025-03-09 21:07:07,936 INFO: Connexion r√©ussie pour l'utilisateur: nadim.benkrouidem@gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:115]
2025-03-09 21:07:18,946 INFO: D√©but de l'upload [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:188]
2025-03-09 21:07:18,949 INFO: Upload vers UserImg/nadim.benkrouidem_at_gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:206]
2025-03-09 21:07:19,255 INFO: Analyse de l'image avec l'IA [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:222]
2025-03-09 21:07:19,701 INFO: Animal identifi√©: raton_laveur avec une confiance de 0.837 [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:237]
2025-03-09 21:33:05,239 INFO: D√©marrage de l'application [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:71]
2025-03-09 21:33:20,666 INFO: Connexion r√©ussie pour l'utilisateur: nadim.benkrouidem@gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:115]
2025-03-09 21:33:32,333 INFO: D√©but de l'upload [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:188]
2025-03-09 21:33:32,337 INFO: Upload vers UserImg/nadim.benkrouidem_at_gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:206]
2025-03-09 21:33:32,656 INFO: Analyse de l'image avec l'IA [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:222]
2025-03-09 21:33:32,830 INFO: Animal identifi√©: Chien avec une confiance de 0.86 [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:237]
2025-03-09 21:39:00,822 INFO: D√©marrage de l'application [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:71]
2025-03-09 21:39:26,753 INFO: Connexion r√©ussie pour l'utilisateur: nadim.benkrouidem@gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:115]
2025-03-19 12:44:56,311 INFO: D√©marrage de l'application [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:71]
2025-03-19 12:45:04,901 INFO: Connexion r√©ussie pour l'utilisateur: nadim.benkrouidem@gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:115]
2025-03-19 12:45:14,233 INFO: D√©but de l'upload [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:188]
2025-03-19 12:45:14,237 INFO: Upload vers UserImg/nadim.benkrouidem_at_gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:206]
2025-03-19 12:45:14,680 INFO: Analyse de l'image avec l'IA [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:225]
2025-03-19 12:45:15,416 INFO: Animal identifi√©: Chat avec une confiance de 0.9909999999999999 [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:242]
2025-03-19 12:45:47,803 INFO: D√©but de l'upload [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:188]
2025-03-19 12:45:47,807 INFO: Upload vers UserImg/nadim.benkrouidem_at_gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:206]
2025-03-19 12:45:48,237 INFO: Analyse de l'image avec l'IA [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:225]
2025-03-19 12:45:48,584 INFO: Animal identifi√©: Rat avec une confiance de 0.9169999999999999 [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:242]
2025-03-19 12:47:26,172 INFO: D√©marrage de l'application [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:71]
2025-03-19 12:47:34,117 INFO: Connexion r√©ussie pour l'utilisateur: nadim.benkrouidem@gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:115]
2025-03-19 12:48:06,645 INFO: Connexion r√©ussie pour l'utilisateur: nadim.benkrouidem@gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:115]
2025-03-19 12:48:18,222 INFO: D√©but de l'upload [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:188]
2025-03-19 12:48:18,227 INFO: Upload vers UserImg/nadim.benkrouidem_at_gmail.com [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:206]
2025-03-19 12:48:18,607 INFO: Analyse de l'image avec l'IA [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:225]
2025-03-19 12:48:18,829 INFO: Animal identifi√©: Puma avec une confiance de 0.9309999999999999 [in /Users/nadimbenkrouidem/Desktop/git_MSPR_6.1/python_file/app.py:242]
2025-05-13 11:39:12,347 INFO: DÈmarrage de l'application [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:71]
2025-05-13 11:39:32,643 ERROR: Erreur de connexion: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:123]
2025-05-13 11:39:40,006 ERROR: Erreur de connexion: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:123]
2025-05-13 11:40:02,916 ERROR: Erreur de connexion: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:123]
2025-05-13 11:40:40,262 ERROR: Erreur de connexion: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:123]
2025-05-13 11:41:21,833 ERROR: Erreur lors de l'inscription: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:166]
2025-05-13 11:41:49,489 ERROR: Erreur lors de l'inscription: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:166]
2025-05-13 11:42:01,197 ERROR: Erreur de connexion: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:123]
2025-05-13 11:42:05,048 ERROR: Erreur de connexion: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:123]
2025-05-13 12:05:00,136 INFO: DÈmarrage de l'application [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:71]
2025-05-13 12:06:46,853 INFO: DÈmarrage de l'application [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:71]
2025-05-13 12:21:40,870 INFO: DÈmarrage de l'application [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:71]
2025-05-13 12:24:14,874 INFO: DÈmarrage de l'application [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:71]
2025-05-13 12:29:13,450 INFO: DÈmarrage de l'application [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:71]
2025-05-13 12:29:49,817 ERROR: Erreur de connexion: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:123]
2025-05-13 12:29:52,386 ERROR: Erreur de connexion: [Errno 11001] getaddrinfo failed [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:123]
2025-05-13 12:47:13,792 INFO: DÈmarrage de l'application [in C:\MSPR\MSPR-6.1\MSPR-6.1-Nadim-Dev\MSPR-6.1-Nadim-Dev\python_file\app.py:71]
